node/bindnode/repr.go

```diff
@@ -473,16 +500,31 @@ func (w *_assemblerRepr) AssignNull() error {
 	}
 }
 
-func (w *_assemblerRepr) AssignBool(bool) error {
-	panic("TODO: AssignBool")
+func (w *_assemblerRepr) AssignBool(b bool) error {
+	switch stg := reprStrategy(w.schemaType).(type) {
+	case nil:
```

Comment:
I'm not sure I understand what `nil` is meaning in these blocks?

Reply:
it means "no representation strategy". essentially, I want to panic if someone uses a method with a repr strategy I do not yet support.

one open question is what AssignBool should do on a known repr strategy where it doesn't make sense, like a struct with stringjoin. I guess we should return a "wrong kind" error then.

Reply:
Yep, "wrong kind" error would be more correct.

Reply:
ACK, I'll handle this in a future PR